tests/test_enterprise/api/test_views.py

```diff
@@ -2401,3 +2401,77 @@ def test_enterprise_customer_catalogs_response_formats(self):
 
         response_xml = self.client.get('/enterprise/api/v1/enterprise_catalogs.xml')
         self.assertTrue(response_xml['content-type'] == 'application/xml; charset=utf-8')
+
+    @mock.patch('django.core.mail.send_mail')
+    @ddt.data(
```

Comment:
Should we also include a use case for the excluding the optional `number_of_codes` parameter?

Reply:
Test added